packages/pieces/schedule/src/lib/triggers/every-x-minutes.trigger.ts

```diff
@@ -1,35 +1,47 @@
-import { Property, TriggerStrategy, Validators, createTrigger } from "@activepieces/pieces-framework";
+import {
+  Property,
+  TriggerStrategy,
+  createTrigger,
+} from '@activepieces/pieces-framework';
 
 export const everyXMinutesTrigger = createTrigger({
-    name: 'every_x_minutes',
-    displayName: 'Every X Minutes',
-    description: 'Triggers the current flow every X minutes',
-    type: TriggerStrategy.POLLING,
-    sampleData: {},
-    props: {
-        minutes: Property.Number({
-            displayName: "Minutes",
-            required: true,
-            defaultValue: 1,
-            validators: [Validators.minValue(1)]
-        })
-    },
-    onEnable: async (ctx) => {
-        const cronExpression = `*/${ctx.propsValue.minutes} * * * *`
-        ctx.setSchedule({
-            cronExpression: cronExpression,
-            timezone: 'UTC'
-        });
-    },
-    run(ctx) {
-        const cronExpression = `*/${ctx.propsValue.minutes} * * * *`
-        return Promise.resolve([{
-            cron_expression: cronExpression,
-            timezone: 'UTC'
-        }]);
-    },
-    onDisable: async () => {
-        console.log('onDisable');
-    }
-
+  name: 'every_x_minutes',
+  displayName: 'Every X Minutes',
+  description: 'Triggers the current flow every X minutes',
+  type: TriggerStrategy.POLLING,
+  sampleData: {},
+  props: {
+    minutes: Property.StaticDropdown({
+      displayName: 'Minutes',
+      description: 'Valid value between 1 to 59.',
+      required: true,
+      defaultValue: 1,
+      options: {
+        disabled: false,
+        options: Array.from({ length: 59 }, (_, index) => ({
+          label: `${index + 1} minutes`,
```

Comment:
Nit: this will produce '1 minutes' which is not correct

Reply:
fixed it.